drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -46,7 +52,11 @@
         </div>
         <div class="row">
             <label>{{i18n "Background image URL (alternative to the text)"}}</label>
```

Comment:
I think we should actually just call this field "Image URL" since it is not used as a background - it replaces the text.

In fact, now that I think about it, we could consider using the text (`displayName`) as the description when an image is present, rather than needing a separate field. You could just say "If you put an image, it will be used instead of the text above. The text above should describe the image so that the exercise can be completed by users who are not able to see the image."

But I'm also fine with keeping the description separate - it's probably cleaner conceptually - and "Text" should be an input while "Description" a textarea so that's another reason to leave it separate.